docs/auth/Single Sign-On via Device Browser.adoc

```diff
@@ -0,0 +1,112 @@
+== Single Sign-On (SSO) via the Device Browser
```

Comment:
Not sure if we need to mention `via the Device Browser` in the header. I think this doc is about how to perform SSO using our IDM SDK. Let's make it obvious.

Reply:
That makes sense, I'll update the heading